apps/ledger-live-desktop/src/renderer/screens/dashboard/index.tsx

```diff
@@ -75,13 +70,14 @@ export default function DashboardPage() {
     },
     [hiddenNftCollections, shouldFilterTokenOpsZeroAmount],
   );
+
   return (
     <>
       <TopBannerContainer>
         <ClearCacheBanner />
         <CurrencyDownStatusAlert currencies={currencies} hideStatusIncidents />
       </TopBannerContainer>
-      {showCarousel ? <Carousel /> : null}
+      <PortfolioContentCards variant={ABTestingVariants.variantA} />
```

Comment:
Not sure to understand properly but variant value should come from Firebase or something like that if we what to ensure A/B Testing?